build-java/action.yml

```diff
@@ -82,6 +82,11 @@ runs:
         path: jasper-report
         token: ${{ inputs.token }}
 
+    - uses: actions/checkout@v3
+      if: ${{ inputs.jasperBuild != 'true' }}
```

Comment:
will this run twice and is there another way to set the condition to provide one of two alternatives?

Reply:
The step directly above is for jasper-report, not the repo this action is building. The one at the very top is the one for the repo being built.